ocviapy/__init__.py

```diff
@@ -596,6 +606,8 @@ def __init__(self, namespace, restype, name, watch_owned=False, watcher=None):
             )
 
     def _check_owned_resources(self, resource):
+        if resource.kind == "pod" and resource.image_pull_error:
```

Comment:
If you go with the above approach, you can remove the first part of this `if`:

```suggestion
        if resource.image_pull_error:
```

Reply:
Added in last push